usaspending_api/database_scripts/etl/award_delta_view.sql

```diff
@@ -20,27 +16,13 @@ SELECT
   vw_es_award_search.award_amount,
   vw_es_award_search.total_subsidy_cost,
   vw_es_award_search.total_loan_value,
-  a.update_date,
+  vw_es_award_search.update_date,
 
   vw_es_award_search.recipient_name,
   vw_es_award_search.recipient_unique_id,
-  recipient_profile.recipient_hash,
-  CASE
-    WHEN recipient_profile.recipient_hash IS NULL or recipient_profile.recipient_levels IS NULL
-      THEN
-        CONCAT(
-          '{"name":"', vw_es_award_search.recipient_name,
-          '","unique_id":"', vw_es_award_search.recipient_unique_id,
-          '","hash":"","levels":""}'
-        )
-    ELSE
-      CONCAT(
-        '{"name":"', vw_es_award_search.recipient_name,
-        '","unique_id":"', vw_es_award_search.recipient_unique_id,
-        '","hash":"', recipient_profile.recipient_hash,
-        '","levels":"', recipient_profile.recipient_levels, '"}'
-      )
-  END AS recipient_agg_key,
+  vw_es_award_search.recipient_hash,
+  vw_es_award_search.recipient_levels,
```

Comment:
is `recipient_levels` needed in the view?

Reply:
No, it looks like it was only previously included to build the agg keys. I will remove it from the mvs, django model, and delta view